[2024-11-04T20:40:04.492+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-04T20:40:04.594+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag_v1.print_hello scheduled__2024-11-04T20:30:00+00:00 [queued]>
[2024-11-04T20:40:04.711+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag_v1.print_hello scheduled__2024-11-04T20:30:00+00:00 [queued]>
[2024-11-04T20:40:04.713+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-04T20:40:04.740+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): print_hello> on 2024-11-04 20:30:00+00:00
[2024-11-04T20:40:04.758+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-04T20:40:04.759+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_dag_v1', 'print_hello', 'scheduled__2024-11-04T20:30:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp9w739yp0']
[2024-11-04T20:40:04.764+0000] {standard_task_runner.py:72} INFO - Started process 239 to run task
[2024-11-04T20:40:04.765+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask print_hello
[2024-11-04T20:40:04.904+0000] {task_command.py:467} INFO - Running <TaskInstance: test_dag_v1.print_hello scheduled__2024-11-04T20:30:00+00:00 [running]> on host f3c6c1109f4c
[2024-11-04T20:40:05.119+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_dag_v1' AIRFLOW_CTX_TASK_ID='print_hello' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T20:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-04T20:30:00+00:00'
[2024-11-04T20:40:05.121+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-04T20:40:05.199+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-04T20:40:05.210+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, World!"']
[2024-11-04T20:40:05.235+0000] {subprocess.py:86} INFO - Output:
[2024-11-04T20:40:05.271+0000] {subprocess.py:93} INFO - Hello, World!
[2024-11-04T20:40:05.272+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-04T20:40:05.323+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-04T20:40:05.324+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_dag_v1, task_id=print_hello, run_id=scheduled__2024-11-04T20:30:00+00:00, execution_date=20241104T203000, start_date=20241104T204004, end_date=20241104T204005
[2024-11-04T20:40:05.383+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-04T20:40:05.412+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-04T20:40:05.418+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
